src/test/java/calculator/OperandTest.java

```diff
@@ -0,0 +1,46 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class OperandTest {
+
+    @DisplayName("Operand 생성 성공")
+    @Test
+    void createSuccess() {
+        //given:
+        String source = "10";
+        //when:
+        Operand operand = new Operand(source);
+        //then:
+        assertThat(operand).isNotNull();
+    }
```

Comment:
생성자를 이용하면 항상 객체를 생성하게 되므로 `null` 일 수 없을 것 같은데요.
다음과 같이 생성을 할 때 예외를 던지지 않는다는 의미로 검증을 하는 방향은 어떻게 생각하시나요?
```suggestion
    void createSuccess() {
        //given:
        String source = "10";
        //when, then:
        assertThatNoException().isThrownBy(() -> new Operand(source));
    }
```

Reply:
항상 이 부분을 고민했었고, 안그래도 이번 step 2 피드백이 오면 질문 드리려고 했던 부분이었습니다 !
이 부분은 지난번 step1에서 한가지 테스트는 한가지 기능만 테스트하도록 하자! 라는 피드백 내용과 관련이 있는 부분이었는데요.
생성자에 대한 테스트만 하고 싶은데.. 꼭 생성된 이후 특정 값이 잘 들어있는지 확인해야 하는게 바람직할까??
에 대한 고민이었습니다.
assertThatNoException().isThrownBy 라는 메서드로, 내가 의도한 대로 잘 생성 되었어! 라는 내용을 검증할 수 있겠군요!!
jupiter에서 굉장히 많은 api들을 제공하고 있었는데, 제대로 활용하지 못한 것 같아 부끄럽네요 ㅎㅎ
피드백 주신 내용 적극 수용하여 다음 과정에서도 꼭 참고하도록 하겠습니다!! 
감사합니다 😊❤️

Reply:
피드백을 수용해주셔서 감사합니다 🙇‍♂️🙇‍♂️🙇‍♂️
그런데 제가 사용한 것은 assertj 이기 때문에 jupiter 와 차이가 있을 수 있습니다!!
참고만해주세요!!